docs/source/algorithms/PolarizationCorrectionFredrikze-v1.rst

```diff
@@ -9,28 +13,43 @@
 
-Algorithm is based on the paper Fredrikze, H, et al. "Calibration of a polarized neutron reflectometer" Physica B 297 (2001).
+The algorithm is based on the paper Fredrikze, H, et al. "Calibration of a polarized neutron reflectometer" Physica B 297 (2001).
+
+Polarizer and Analyzer efficiencies are calculated and used to perform an intensity correction on the input workspace. The input workspace(s) are in units of wavelength (inverse angstroms).
+
+In the ideal case :math:`P_{p} = P_{a} = A_{p} = A_{a} = 1`.
+
+:math:`\rho = \frac{P_{a}}{P_{p}}` where rho is bounded by, but inclusive of 0 and 1. Since this ratio is wavelength dependent, rho is a polynomial, which is expressed as a function of wavelength. For example:
+:math:`\rho(\lambda) =\sum\limits_{i=0}^{i=2} K_{i}\centerdot\lambda^i`, can be provided as :math:`K_{0}, K_{1}, K_{2}`.
 
-Polarizer and Analyzer efficiencies are calculated and used to perform an intensity correction on the input workspace. The input workspace(s) are in units of wavelength
-inverse angstroms.
+:math:`\alpha = \frac{A_{a}}{A_{p}}` where alpha is bounded by, but inclusive of 0 and 1. Since this ratio is wavelength dependent, alpha is a polynomial, which is expressed as a function of wavelength. For example:
+:math:`\alpha(\lambda) =\sum\limits_{i=0}^{i=2} K_{i}\centerdot\lambda^i`, can be provided as :math:`K_{0}, K_{1}, K_{2}`.
 
-In the ideal case :math:`P_{p} = P_{a} = A_{p} = A_{a} = 1`
+Spin State Configurations
+#########################
+The algorithm directly uses spin states to define the measurement setup. The spin states are specified using two properties:
 
-:math:`\rho = \frac{P_{a}}{P_{p}}` where rho is bounded by, but inclusive of 0 and 1.
-Since this ratio is wavelength dependent, rho is a polynomial, which is expressed as a function of wavelength. For example:
-:math:`\rho(\lambda) =\sum\limits_{i=0}^{i=2} K_{i}\centerdot\lambda^i`, can be provided as :math:`K_{0}, K_{1}, K_{2}`
+**Input Spin State Order:**
+This property determines the order of the spin states in the input workspace group
 
-:math:`\alpha = \frac{A_{a}}{A_{p}}` where alpha is bounded by, but inclusive of 0 and 1.
-Since this ratio is wavelength dependent, alpha is a polynomial, which is expressed as a function of wavelength. For example:
-:math:`\alpha(\lambda) =\sum\limits_{i=0}^{i=2} K_{i}\centerdot\lambda^i`, can be provided as :math:`K_{0}, K_{1}, K_{2}`
+- :literal:`'pp, pa, ap, aa'`
+   Full polarization corrections: both parallel, parallel-anti-parallel, anti-parallel-parallel, both  anti-parallel. Four input workspaces are required. The spin states can be provided in any order and should match the order of the workspaces in the input group.
+
+- :literal:`'p, a'`
+   PNR polarization corrections:  parallel, anti-parallel. Two input workspaces are required. The spin states can be provided in any order and should match the order of the workspaces in the input group.
```

Comment:
Can we make clear here that `PA` is Full polarization corrections and what `PNR` stands for?